### ISSUE TITLE
Failed to Launch JVM error

### ISSUE THREAD

**Issue by Playrex** [NyCodeGHG/omsi-launcher#77]
Hi, 
When I launch omsi-launcher, two errors appear : the first is "Failed to launch JVM" and the second one is "Child process exited with code 1"

Is there a way to solve these errors ? 

![image](https://user-images.githubusercontent.com/112385262/187166159-f0abb7b8-0940-407c-96a2-44848dfd7d81.png)
![image](https://user-images.githubusercontent.com/112385262/187166575-e5a1e67f-2be3-4609-a7cf-9ea4e3ec1b7d.png)

**Comment by DRSchlaubi** [comment]
Could you please attach your `%APPDATA%/omsi-launcher/logs/latest.log` file

**Comment by DRSchlaubi** [comment]
Could you try downloading the latest release directly from here? https://github.com/NyCodeGHG/omsi-launcher/releases/download/v0.7.0/omsi-launcher-0.7.0.msi

**Comment by DRSchlaubi** [comment]
Looks like 0.6.4 is broken, 0.7.0 can be downloaded from the link above until https://github.com/microsoft/winget-pkgs/pull/75387 gets merged

**Comment by Playrex** [comment]
I have only one file into logs folder. It's logFile.log
[logFile.log](https://github.com/NyCodeGHG/omsi-launcher/files/9446840/logFile.log)

**Comment by Playrex** [comment]
With the latest release I've the same error :(

Maybe I did something wrong...

**Comment by DRSchlaubi** [comment]
DId you try downloading 0.7.0

**Comment by Playrex** [comment]
Yes, I downloaded and installed it with your link but nothing change

**Comment by DRSchlaubi** [comment]
Could you please send us your
`<instancesFolder>/instances.json`
And your 
`%APPDATA%/omsi-launcher/config.json`

**Comment by DRSchlaubi** [comment]
Delete the config.json and try again

**Comment by Playrex** [comment]
Okay

I did it and the config.json is recreated and the errors appears again just after.

**Comment by DRSchlaubi** [comment]
Delete the config.json again and reinstall

**Comment by Playrex** [comment]
The same :(

**Comment by DRSchlaubi** [comment]
Unfortunately, that's all Ican do. Maybe @NyCodeGHG has an idea

**Comment by NyCodeGHG** [comment]
i have no idea why this could happen. i'm sorry

**Comment by DRSchlaubi** [comment]
The last thing you could try is
- Run `winget install EclipseAdoptium.Temurin.18.JDK`
- Download this: https://github.com/NyCodeGHG/omsi-launcher/archive/refs/tags/v0.7.0.zip
- Unzip it
- open a terminal (in the unzipped folder)
- run `gradlew runDistributable`

**Comment by Luca00711** [comment]
I have the same problem. 

And I found the problem in the souce code.


The Problem is the lateinit property config in `config/ConfigReader.kt: 23`
When there is no LauncherConfigFile the function gets returned in line `32` then the error happends cause the property config is not going to initialize.

With this error the config.json file get created, but just empty (EOF). So at the next start only the error `Failed to launch JVM` appears cause the json is not parseable.

**Comment by DRSchlaubi** [comment]
Currently, we're still waiting for Sectigo to renew our certificate, as they're still processing our request after 2 weeks now

**Comment by NyCodeGHG** [comment]
@Playrex you can try this version, but windows will cry because it's not signed
https://cdn.discordapp.com/attachments/809316998689456168/1022134643216109659/omsi-launcher-0.7.1.msi

**Comment by Playrex** [comment]
Hi, 
Okay, I will try this evening. I will tell you if it works.

**Comment by Playrex** [comment]
I can't access the URL because I've this error
![image](https://user-images.githubusercontent.com/112385262/191671206-0e89b660-a450-411e-b0a2-8d8c6d9430f6.png)

**Comment by DRSchlaubi** [comment]
The update is now out on the usual channels as well

**Comment by Playrex** [comment]
Okay, 
So, now I don't have the JVM error but I still have the Child process error
![image](https://user-images.githubusercontent.com/112385262/192133872-033b2fb4-cacb-4879-887a-5e1890184fc3.png)

**Comment by DRSchlaubi** [comment]
could you delete the config.json again?

**Comment by Playrex** [comment]
Okay,
I did it and the error continue to appear. But the config.json is no longer created when I start the program.

**Comment by DRSchlaubi** [comment]
> The last thing you could try is
> 
> * Run `winget install EclipseAdoptium.Temurin.18.JDK`
> * Download this: https://github.com/NyCodeGHG/omsi-launcher/archive/refs/tags/v0.7.0.zip
> * Unzip it
> * open a terminal (in the unzipped folder)
> * run `gradlew runDistributable`

Try this
